src/components/dialog/dialog.spec.js

```diff
@@ -818,7 +818,7 @@ describe('$mdDialog', function() {
 
       it('should not wrap raw content with md-dialog', inject(function($mdDialog, $rootScope) {
 
-        var template = '<div id="rawContent">Hello</div>';
+        var template = '<md-dialog id="rawContent">Hello</md-dialog>';
```

Comment:
@ThomasBurleson With a `<div>`, the test fails because it's trying to find an `<md-dialog>` and it can't, so we get an error on `element[0].parentNode`: `cannot find parentNode of undefined`.

@gkalpak I think it's definitely similar, but we do assume that the raw content will have a dialog. I think the test was passing erroneously because we were swallowing an error that in 1.6 was getting through.

In fact, the test description talks about the `<md-dialog>`, so I think this was just an oversight.

How do you think we could make this better or more clear? Do you mean adding some documentation?